operator/v1/types_console.go

```diff
@@ -174,6 +178,15 @@ type DeveloperConsoleCatalogCategoryMeta struct {
 	Tags []string `json:"tags,omitempty"`
 }
 
+type AccessReviewResources struct {
+	// kind is a string value representing the REST resource.
+	// +required
```

Comment:
This isn't the correct tag, it should be:
```suggestion
	// +kubebuilder:validation:Resource
```